### DISCUSSION TITLE
Shape of tagger with pT bin for ZV MC Jialin

### DISCUSSION THREAD

**Discussion by jialin-guo1** [ram1123/2l2Q_limitSettingTool#46] [Comments_HIG_23_001] [answered]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/c07ba100-78ee-4844-8cf8-4852e846922d)

**Comment by ram1123** [comment]
This is after cut.

We need before cut the shapes in the bins of pT

**Reply by jialin-guo1** [reply]
|pT bin | 200-400Gev  | 400-600Gev | 600-Inf |
|---|---|---|---|
| distribution  | ![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/4105d155-0f3c-4218-9599-4cc56d1db4b3) | ![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/0eaa500f-44fe-4e23-94a2-24829c670455) | ![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/f961d5e4-2c65-4f9a-8775-f40b16e63afb) |

**Reply by ram1123** [reply]
Hi @jialin-guo1 ,

Can you make all three together on one canvas?

**Reply by jialin-guo1** [reply, marked as answer]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/4b30bea2-82d3-4a77-a051-1ef0ed4d9ef0)

yes sure
